cpp/ql/src/Likely Bugs/Memory Management/SuspiciousCallToStrncat.qhelp

```diff
@@ -4,7 +4,11 @@
 <qhelp>
 <overview>
 <p>The standard library function <code>strncat</code> appends a source string to a target string. 
-The third argument defines the maximum number of characters to append and should be less than or equal to the remaining space in the destination buffer. Calls of the form <code>strncat(dest, src, strlen(dest))</code> or <code>strncat(dest, src, sizeof(dest))</code> set the third argument to the entire size of the destination buffer. Executing a call of this type may cause a buffer overflow unless the buffer is known to be empty. Buffer overflows can lead to anything from a segmentation fault to a security vulnerability.</p>
+The third argument defines the maximum number of characters to append and should be less than or equal to the remaining space in the destination buffer.
```

Comment:
All these sentences are in the same paragraph. Perhaps consider breaking this content into 2 or 3 paragraph to improve readability? Feel free to ignore if you don't agree.

![one-para](https://user-images.githubusercontent.com/42146119/119467047-181adb00-bd3d-11eb-9e46-465d4ef7c0df.png)

Reply:
Oh! Good point. Fixed in 5382ef7761c32025a5a63344934ae51330ddbf6d.